src/xrCore/xrMemory.h

```diff
@@ -12,19 +11,18 @@
 //#include "tbb/tbbmalloc_proxy.h" // Xottab_DUTY: works bad, disabled it..
 #endif
 
+#if defined(WINDOWS)
+#define NO_TBB_MALLOC
```

Comment:
Why you enable it only for windows? Does the Linux port has some problems with it?

Reply:
No. On Linux this define can be enabled using CMake.

Reply:
On windows need use Common.props